cpu/lm4f120/include/periph_cpu.h

```diff
@@ -115,6 +115,27 @@ typedef enum {
     ADC_RES_16BIT = 0xd00,            /**< not supported by hardware */
 } adc_res_t;
 
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF 4
```

Comment:
It's not really a rule as per the coding conventions, but it's commonly used throughout the code base. The problem is that when the macro gets more complex over time, it is easier to forget the parens if they weren't there before